mage_ai/api/presenters/PipelinePresenter.py

```diff
@@ -66,6 +66,14 @@ async def present(self, **kwargs):
                     FeatureUUID.DATA_INTEGRATION_IN_BATCH_PIPELINE,
                 )
 
+            include_variables_dir = query.get('include_variables_dir', [False])
```

Comment:
You don't necessarily need to add these two params.
Including variables_dir and remote_variables_dir don't need extra IO or DB operations. So you can just include them in the base response payload.

Reply:
Ah ok. Updated.